parmed/tools/actions.py

```diff
@@ -2927,6 +2927,16 @@ def __repr__(self):
                    len(self.parm.atoms), len(self.parm.residues))
         )
 
+        _rset = set(res.name for res in self.parm.residues).__str__() + '\n'
+        _rcount = Counter(res.name for res in self.parm.residues).__str__() + '\n'
```

Comment:
why not `str(<blah>)` instead of `<blah>.__str__()`?  The former looks a bit more natural.